curriculum/challenges/english/25-front-end-development/review-javascript-audio-and-video/6723cf27c6e9a0c3f3041385.md

```diff
@@ -9,7 +9,82 @@ dashedName: review-javascript-audio-and-video
+- **`play()` Method**: This method is used with the `<audio>` or `<video>` elements to begin playback for the media.
+
+```js
+const audio = document.getElementById('audio');
+
+// Starts playing the audio
+audio.play();  
+```
+
+- **`pause()` Method**: This method is used with the `<audio>` or `<video>` elements to pause playback for the media.
+
+```js
+function pauseAudio() {
+  const audio = document.getElementById('myAudio');
+  audio.pause();  // Pauses the audio playback
+}
+```
+
+- **`addTextTrack()` Method**: This method allows you to specify a text track to associate with the media element - which is especially helpful for adding subtitles to a video.
+- **`fastSeek()` Method**: This method allows you to move the playback position to a specific time within the media.
+
+## Different Audio and Video Formats
+
+- **MIME type**: A MIME type, standing for Multipurpose Internet Mail Extensions, is a standardized way to programmatically indicate a file type. The MIME type can tell an application, such as your browser, how to handle a specific file. In the case of audio and video, the MIME type indicates it is a multimedia format that can be embedded in the web page.
+- **`source` Element**: This is used to specify a file type and source - and can include multiple different types by using multiple source elements. When you do this, the browser will determine the best format to use for the user’s current environment.
+- **MP3**: This is a type of digital file format used to store music, audio, or sound. It's a compressed version of a sound recording that makes the file size smaller, so it's easier to store and share. An MP3 file has the MIME type audio/mp3
+- **MP4**: An MP4 is a type of digital file format used to store video and audio. It serves as a container that holds both the video (images) and the sound (music or speech) in one file. An MP4, can have the MIME type audio/mp4 OR video/mp4, depending on whether it’s a video file or audio-only. 
+
+## codecs
+
+- **Definition**: A codec, short for "encoder/decoder", is an algorithm or software that can convert audio and video between analogue and digital formats. Codecs can be specified as part of the MIME type. The basic syntax to define a codec is to add a semi-colon after the media type, then `codecs=` and the codec.
+
+## HTMLMediaElement API
+
+- **Definition**: The `HTMLMediaElement` API is used to control the behavior of audio and video elements on your page. It extends the base `HTMLElement` interface, so you have access to the base properties as well as these helpful methods. Examples of these methods include `play()`, `stop()`, and `pause()`.
```

Comment:
I don't find any `stop()` method. I guess the idea was to include `fastSeek()`? Since it was mentioned before
```suggestion
- **Definition**: The `HTMLMediaElement` API is used to control the behavior of audio and video elements on your page. It extends the base `HTMLElement` interface, so you have access to the base properties as well as these helpful methods. Examples of these methods include `play()`, `fastSeek()`, and `pause()`.
```